### ISSUE TITLE
IDE Fatal Errors

### ISSUE THREAD

**Issue by JaPawel** [gmatyszczak/screen-generator-plugin#2]
```
Read access is allowed from event dispatch thread or inside read-action only (see com.intellij.openapi.application.Application.runReadAction())
Details: Current thread: Thread[ApplicationImpl pooled thread 5,4,Idea Thread Group] 1115660594
; dispatch thread: false; isDispatchThread(): false
SystemEventQueueThread: Thread[AWT-EventQueue-0,6,Idea Thread Group] 1312792738

com.intellij.openapi.application.impl.ApplicationImpl$NoReadAccessException
	at com.intellij.openapi.application.impl.ApplicationImpl.assertReadAccessAllowed(ApplicationImpl.java:1085)
	at com.intellij.openapi.fileEditor.impl.FileDocumentManagerImpl.getDocument(FileDocumentManagerImpl.java:172)
	at com.intellij.psi.impl.PsiDocumentManagerBase.getDocument(PsiDocumentManagerBase.java:182)
	at com.intellij.ui.LanguageTextField.createDocument(LanguageTextField.java:96)
	at com.intellij.ui.LanguageTextField$SimpleDocumentCreator.createDocument(LanguageTextField.java:79)
	at com.intellij.ui.LanguageTextField.<init>(LanguageTextField.java:63)
	at com.intellij.ui.LanguageTextField.<init>(LanguageTextField.java:47)
	at ui.settings.CodePanel.createLanguageTextField(CodePanel.kt:62)
	at ui.settings.CodePanel.createLanguageTextField$default(CodePanel.kt:61)
	at ui.settings.CodePanel.<init>(CodePanel.kt:16)
	at ui.settings.SettingsPanel.<init>(SettingsPanel.kt:36)
	at ui.settings.SettingsViewImpl.<init>(SettingsViewImpl.kt:16)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.picocontainer.defaults.InstantiatingComponentAdapter.newInstance(InstantiatingComponentAdapter.java:193)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.doGetComponentInstance(CachingConstructorInjectionComponentAdapter.java:99)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.instantiateGuarded(CachingConstructorInjectionComponentAdapter.java:76)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getComponentInstance(CachingConstructorInjectionComponentAdapter.java:59)
	at com.intellij.openapi.extensions.AbstractExtensionPointBean.instantiate(AbstractExtensionPointBean.java:80)
	at com.intellij.openapi.options.ConfigurableEP$ClassProducer.createElement(ConfigurableEP.java:334)
	at com.intellij.openapi.options.ConfigurableEP.createConfigurable(ConfigurableEP.java:260)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.createConfigurable(ConfigurableWrapper.java:57)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getConfigurable(ConfigurableWrapper.java:116)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.cast(ConfigurableWrapper.java:97)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getDisplayName(ConfigurableWrapper.java:137)
	at com.intellij.openapi.options.ex.Weighted.lambda$static$0(Weighted.java:35)
	at java.util.TimSort.binarySort(TimSort.java:296)
	at java.util.TimSort.sort(TimSort.java:221)
	at java.util.Arrays.sort(Arrays.java:1512)
	at java.util.ArrayList.sort(ArrayList.java:1456)
	at java.util.Collections.sort(Collections.java:175)
	at com.intellij.openapi.options.ex.SortedConfigurableGroup.buildConfigurables(SortedConfigurableGroup.java:53)
	at com.intellij.openapi.options.SearchableConfigurable$Parent$Abstract.getConfigurables(SearchableConfigurable.java:94)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.collect(ShowSettingsUtilImpl.java:89)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.getConfigurables(ShowSettingsUtilImpl.java:80)
	at com.intellij.ide.actions.SearchEverywhereAction$CalcThread.checkModelsUpToDate(SearchEverywhereAction.java:1980)
	at com.intellij.ide.actions.SearchEverywhereAction$CalcThread.run(SearchEverywhereAction.java:1373)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:314)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

Android Studio 3.3.2
Build #AI-182.5107.16.33.5314842, built on February 16, 2019
JRE: 1.8.0_152-release-1248-b01 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.3

Steps to reproduce:
Try to open `Search Everywhere` (double shift) and type anything - this is a situation, when I am always getting this error (tested in few Android Projects).

I had the similar error [here](https://github.com/abeade/fcm-push-plugin/issues/1) - now it's fixed.

<img width="858" alt="error" src="https://user-images.githubusercontent.com/8151020/54883611-a21e3f00-4e67-11e9-991e-0df6fe39d795.png">

**Comment by gmatyszczak** [comment]
@JaPawel Thanks for detailed report and tips on how to fix it! :) You can check version 1.0.1, it should be fixed now.

**Comment by JaPawel** [comment]
@gmatyszczak Thanks 👍
